packages/radix-vue/src/DismissableLayer/utils.ts

```diff
@@ -122,13 +125,12 @@ export function usePointerDownOutside(
      * });
      */
     const timerId = window.setTimeout(() => {
-      ownerDocument.addEventListener('pointerdown', handlePointerDown)
+      ownerDocumentPointerdownCleanup = useEventListener(ownerDocument, 'pointerdown', handlePointerDown)
     }, 0)
 
     cleanupFn(() => {
       window.clearTimeout(timerId)
-      ownerDocument.removeEventListener('pointerdown', handlePointerDown)
-      ownerDocument.removeEventListener('click', handleClickRef.value)
+      ownerDocumentPointerdownCleanup()
```

Comment:
maybe we need to add nullish coalasce `ownerDocumentPointerdownCleanup && ownerDocumentPointerdownCleanup()`

Reply:
Done